src/pages/settings/settings.html

```diff
@@ -34,91 +46,96 @@
         </ion-icon>
-        <ion-label>
-          {{'Alternative currency' | translate}}
-        </ion-label>
+        <ion-label> {{ 'Alternative currency' | translate }} </ion-label>
         <ion-note *ngIf="selectedAlternative" item-end>
           {{ selectedAlternative.name }}
         </ion-note>
       </button>
 
       <button ion-item (click)="openFeePolicyPage()">
         <ion-icon class="item-img settings-icon" item-start>
-          <img src="assets/img/settings-icons/icon-network.svg" width="22">
+          <img src="assets/img/settings-icons/icon-network.svg" width="22" />
         </ion-icon>
-        <ion-label>
-          {{'Bitcoin network fee policy' | translate}}
-        </ion-label>
+        <ion-label> {{ 'Bitcoin network fee policy' | translate }} </ion-label>
       </button>
 
       <button ion-item *ngIf="isCordova" (click)="openLockPage()">
         <ion-icon class="item-img settings-icon" item-start>
-          <img src="assets/img/settings-icons/icon-lock.svg" width="22">
+          <img src="assets/img/settings-icons/icon-lock.svg" width="22" />
         </ion-icon>
-        <ion-label>
-          {{'Lock' | translate}}
-        </ion-label>
+        <ion-label> {{ 'Lock' | translate }} </ion-label>
         <ion-note *ngIf="lockMethod === 'disabled'" item-end>
-          {{'Disabled' | translate}}
+          {{ 'Disabled' | translate }}
         </ion-note>
         <ion-note *ngIf="lockMethod === 'pin'" item-end>
-          {{'PIN' | translate}}
+          {{ 'PIN' | translate }}
         </ion-note>
         <ion-note *ngIf="lockMethod === 'fingerprint'" item-end>
-          {{'Biometric' | translate}}
+          {{ 'Biometric' | translate }}
         </ion-note>
       </button>
 
       <div *ngIf="showBitPayCard && bitpayCardItems && bitpayCardItems[0]">
-        <ion-item-divider>
-          BitPay Visa
-          <sup>&reg;</sup> Card
-        </ion-item-divider>
+        <ion-item-divider> BitPay Visa <sup>&reg;</sup> Card </ion-item-divider>
         <ion-list>
           <button ion-item *ngFor="let card of bitpayCardItems" (click)="openCardSettings(card.id)">
             <ion-icon class="item-img" item-start>
               <img src="assets/img/icon-card.svg" class="icon-card" />
             </ion-icon>
-            <div class="item-title">BitPay Visa&reg; Card ({{card.lastFourDigits}})</div>
-            <div class="item-subtitle">{{card.balance ? (card.balance | currency:card.currencySymbol:2) : 'Add funds to
-                get started'|translate}} {{card.updatedOn
-                ? (' &middot; ' + (card.updatedOn * 1000 | amTimeAgo)) : ''}}</div>
+            <div class="item-title">
+              BitPay Visa&reg; Card ({{ card.lastFourDigits }})
+            </div>
+            <div class="item-subtitle">
+              {{
+                card.balance
+                  ? (card.balance | currency: card.currencySymbol:2)
+                  : ('Add funds to
```

Comment:
not sure this line break will work ok with translate, better leave it on one line